src/MySql/docs/Update-AzMySqlVirtualNetworkRule.md

```diff
@@ -31,30 +31,26 @@ Creates or updates an existing virtual network rule.
 
 ## EXAMPLES
 
-### Example 1: Update MySql Virtual Network Rule by name
```

Comment:
don't find a change in examples/Update-AzMySqlVirtualNetworkRule.md

Reply:
Same as my reply to the later comment. 

Lei I'm little lost here. I didn't change anything for the single server commands... what kind of commits are you referring here...? Could you kindly explain to me? :)